src/browser/convert_options.js

```diff
@@ -274,6 +274,10 @@ module.exports = {
             newOptions.preload = preload;
         }
 
+        if (options.customRequestHeaders !== undefined) {
```

Comment:
This isn't necessary if you just put the window option in this object: https://github.com/HadoukenIO/core/blob/develop/src/browser/convert_options.js#L35

Reply:
This can be ignored. Discussed offline.